src/components/manager/projects/ProjectJsonSchemaForm.js

```diff
@@ -14,6 +14,18 @@ const ajv = new Ajv({
 // Does not actually query over http, the URI is the key to the draft-07 meta-schema
 const validateSchema = ajv.getSchema("http://json-schema.org/draft-07/schema");
 
+const getSchemaTypeOptions = (schemaTypes) => {
+    if (schemaTypes instanceof Object) {
+        return Object.entries(schemaTypes).map(([key, value]) => ({
+            key,
+            value: key,
+            text: value.toUpperCase(),
+        }));
+    } else {
+        return [];
```

Comment:
when is this not an object?

Reply:
I changed instanceof to typeof === "object". The previous way could cause problems in some situations, but this new way makes sure that schemaTypes is always an object and checks for no null. It should be more reliable. Thanks for pointing that out!